### DISCUSSION TITLE
I don't think we should do analitics

### DISCUSSION THREAD

**Discussion by donaldli2020** [Prodigy-Hacking/ProdigyMathGameHacking#1102] [(C) Suggestions]
I think we shouldn't add analitics because of security. I wouldn't want people to know personal info.

**Comment by hostedposted** [comment]
We are soon going to add analytics. We can not get any personal info with analytics.

**Comment by PatheticMustan** [comment]
Usually by analytics, it just means "we have 10 people who used this hack", "this hack has a lot of errors" or whatever. This is mainly to help us figure out which hacks don't work anymore.

**Comment by donaldli2020** [comment]
There's age and location

**Reply by hostedposted** [reply]
bruh location is city and age is based off grade.

We can't get the actual age or location that's impossible to do from prodigy.

**Reply by donaldli2020** [reply]
You can use IP addresses, but websites will think I'm in Quebec, which is not true.

**Reply by donaldli2020** [reply]
This will display your IP
```
$.getJSON("https://api.ipify.org/?format=json", function (response) {
ip = response.ip;
$("#ip").text(response.ip);
window.alert("Your IP is " + ip);
});
```

**Reply by hostedposted** [reply]
An IP address only contains the city.

**Reply by donaldli2020** [reply]
![image](https://user-images.githubusercontent.com/74021676/131933412-0ce0fe71-0c28-4f0f-99de-68fd6e457707.png)
They think I'm in Montreal.

**Comment by afkvido** [comment]
We could just use GitHub analytics for now

**Comment by 00100000** [comment]
This is pretty funny considering your ip is sent to and most likely logged by almost every website you visit, and you probably use Windows + Chrome which are closed source programs that track a lot of extremely specific user data and are allowed to use and sell that data to other companies for targeted advertising. If you're going to complain about minimally invasive tracking that's going to be used for improving the overall experience of using PMGH, you should at least be acting in a way that shows you care about privacy. Unless you have a VPN on all the time and are using FOSS everywhere, you really have to right to complain about this...

I'm not saying you have to be a tails linux + tor user to want basic privacy, but you just come off as pretty ignorant and obnoxious here.
